home/export_whatsapp_templates.py

```diff
@@ -0,0 +1,188 @@
+import copy
+import csv
+import io
+from collections.abc import Iterable, Iterator
+from dataclasses import asdict, astuple, dataclass, fields
+from math import ceil
+
+from django.http import HttpResponse  # type: ignore
+from openpyxl.styles import Border, Color, Font, NamedStyle, PatternFill, Side
+from openpyxl.utils import get_column_letter
+from openpyxl.workbook import Workbook
+from openpyxl.worksheet.worksheet import Worksheet
+from wagtail.query import PageQuerySet  # type: ignore
+
+
+@dataclass
+class ExportRow:
+    name: str = ""
+    category: str = ""
+    quick_replies: str = ""
+    locale: str = ""
+    image: str = ""
+    message: str = ""
+    example_values: str = ""
+    submission_name: str = ""
+    submission_status: str = ""
+    submission_result: str = ""
+
+    @classmethod
+    def headings(cls) -> list[str]:
+        return [f.name for f in fields(cls)]
+
+    def to_dict(self) -> dict[str, str | int]:
+        return asdict(self)
+
+    def to_tuple(self) -> tuple[str | int, ...]:
+        return astuple(self)
+
+
+class WhatsAppTemplateExporter:
+    rows: list[dict[str | int, str | int]]
+
+    def __init__(self, queryset: PageQuerySet):
+        self.rows = []
+        self.queryset = queryset
+
+    def perform_export(self) -> Iterable[ExportRow]:
+        image_link = ""
```

Comment:
This, being outside the loop, will mean that it will be blank, until we come across a template with an image, at which point it will have a value. But then any further templates will keep the value, until another image comes along to replace that value. So if you have templates like:
```
Template(image=None)
Template(image="url1")
Template(image=None)
Template(image="url2")
```
You'll get:
```
""
"url1"
"url1"
"url2"
```
Instead of:
```
""
"url1"
""
"url2"
```

I think we just need to move this into the for loop to fix it.